lib/plugins/slstats/tests/slstats.js

```diff
@@ -82,6 +82,25 @@ describe('SlStats', () => {
       ).to.equal(false);
     });
 
+    it('should throw error if the stats file is not exists', () => {
+      // create a stats-disabled file
```

Comment:
Do we need this `stats-error-file`? If not it would be great if we can get rid of it.